src/ResourceManager/Sql/ChangeLog.md

```diff
@@ -18,7 +18,6 @@
         - Additional information about change #1
 -->
 ## Current Release
+* Added storage properties to threat detection cmdlets
```

Comment:
@yoavrubin Please be specific about which cmdlets are affecetd (please list affected cmdlets)